tests/bud.bats

```diff
@@ -3446,3 +3446,63 @@ _EOF
 
   run_buildah rmi -f ${target}
 }
+
+@test "bud with run should not leave mounts behind cleanup test" {
+  skip_if_in_container
+  run which podman
+  if [[ $status -ne 0 ]]; then
+    skip "podman is not installed"
+  fi
+
+  # Create target dir where we will export tar
+  target=cleanable
```

Comment:
Go ahead and this under $TESTDIR, which is created fresh for each test by the `setup()` function.